wie-geht-studium/vorlesung_allgemein.tex

```diff
@@ -1,12 +1,12 @@
 \section[Lehrveranstaltungen an der Uni]{Lehrveranstaltungen \\an der Uni}
 
 \begin{figure*}[b]
-	\centering{
+    \centering{
     \includegraphics[width=\textwidth]{bilder/computer_lights.png}
 }
 \end{figure*}
 
-Im Gegensatz zur Schule gibt es an der Uni eine Reihe verschiedenartiger Veranstaltungen, in denen der Lehrstoff vermittelt werden soll. Im Wesentlichen sind das: Vorlesungen, Übungen, Seminare und Praktika. Ihr habt im ersten Semester nur mit den ersten beiden zu tun. Diejenigen unter euch, die Physik studieren, werden ab dem zweiten Semester auch Praktika absolvieren. Proseminare werden in Mathe und Info ab dem zweiten Semester angeboten. Beides braucht euch also jetzt noch nicht zu beunruhigen. Ihr findet im Verlauf dieses Abschnitts zu beidem noch einiges an Informationen.
+Im Gegensatz zur Schule gibt es an der Uni eine Reihe verschiedenartiger Veranstaltungen, in denen der Lehrstoff vermittelt werden soll. Im Wesentlichen sind das: Vorlesungen, Übungen, Seminare und Praktika. Ihr habt im ersten Semester nur mit den ersten beiden zu tun. Diejenigen unter euch, die Physik studieren, werden ab dem zweiten Semester auch Praktika absolvieren. Proseminare in Mathe bzw. Bachelorseminare in der Informatik werden ab dem zweiten Semester angeboten. Beides braucht euch also jetzt noch nicht zu beunruhigen. Ihr findet im Verlauf dieses Abschnitts zu beidem noch einiges an Informationen.

```

Comment:
Hier kann man vielleicht schon erklären, dass Bachelorseminare in der Mathematik als Teil der Bachelorarbeit was anderes sind als in der Informatik. Vielleicht in einer Fußnote oder so.